Example/Backpack/View Controllers/ToastViewController.swift

```diff
@@ -44,14 +44,14 @@ class ToastViewController: UIViewController {
         toast.mode = .indeterminate
         toast.labelText = "This is the title"
         toast.detailsLabelText = "Details label text"
-        toast.hide(true, afterDelay: 1.2)
+        toast.hide(true, afterDelay: 5)
```

Comment:
I had to change these to ensure they are visible when the screenshot is captured